### PR TITLE
Update nuget packages

### PR DESCRIPTION
**Pull request by teinarss**

### CODE REVIEW

File: OpenRA.Game/OpenRA.Game.csproj [hunk 1]
```diff
@@ -16,6 +16,6 @@
     <PackageReference Include="Mono.NAT" Version="3.0.1" />
     <PackageReference Include="SharpZipLib" Version="1.3.3" />
     <PackageReference Include="System.Net.Http" Version="4.3.4" />
-    <PackageReference Include="System.Threading.Channels" Version="5.0.0" />
+    <PackageReference Include="System.Threading.Channels" Version="6.0.0-rc.1.21451.13" />
```
Comment (pchote) [inline OpenRA.Game/OpenRA.Game.csproj]: Is it safe to bump this one before we switch to net6?

**Review by pchote — DISMISSED** [review]

File: OpenRA.Game/OpenRA.Game.csproj (hunk 1 above)
Comment (abcdefg30) [reply OpenRA.Game/OpenRA.Game.csproj]: @teinarss ?

Comment (penev92) [reply OpenRA.Game/OpenRA.Game.csproj]: This is going to need another update in a month when .NET 6 releases and is also an experimental build. I say skip it until the final thing is available.

**Comment by teinarss** [comment]
updated

**Review by abcdefg30 — APPROVED** [review]

**Pull request merged**